lib/src/interceptor.dart

```diff
@@ -53,11 +73,12 @@ class Registry {
 
 typedef ExceptionThrower = void Function();
 
-class Interceptor {
+class  Interceptor {
   final RequestMatcher _matcher;
+  final bool generateAuto;
 
   Map<String, String>? replyHeaders;
-  late int statusCode;
+  int? statusCode;
```

Comment:
I tried to avoid making changes like this, but this was necessary, I'm afraid.  

statusCode isn't really late: it genuinely is nullable: it's set only when a method sets it, but otherwise is unset.  That wouldn't be a problem, except methods aren't the only way to register an Interceptor: `throwing` does too.  So, I've changed this so I can copy the Interceptor whether or not it has a method defined.